src/Service/Messenger.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Service;
+
+
+use App\Model\Message;
+
+class Messenger
+{
+    /**
+     * @var SenderInterface[]
+     */
+    protected $senders = [];
+
+    /**
+     * Messenger constructor.
+     * @param SenderInterface[] $senders
+     */
+    public function __construct(array $senders = [])
+    {
+        $this->senders = $senders;
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     */
+    public function send(Message $message)
+    {
+        foreach ($this->senders as $sender) {
```

Comment:
if we rework construct method, we can use return $this->sender?->support()?->send();
Of course need to rework Sender methods - to set $message variable at the initiation.